docs/user-guide/api-mediation/api-mediation-smf.md

```diff
@@ -0,0 +1,83 @@
+# SMF records
+
+API Mediation Layer can issue SMF type 83, 230, or 231 security-related audit records. This requires the generation of a Personal Access Token.
```

Comment:
I think it would be useful to specify the use case for issuing these types of SMF records.
Something like, "You can use SMF records to assist with analyzing configuration scheduling jobs, evaluating data set activity, profiling system resource use and maintaining system security."

Reply:
I agree. `This requires the generation of a Personal Access Token.` is the use case. I would not say it is requirement. It is rather the first use case.

Otherwise, I can approve it.

Reply:
Well, mine original sentence was this one, without any "requires", that @janan07 changed:
https://github.com/zowe/docs-site/blob/0647cd355e42b16bb2516a41696c816e45baeace/docs/user-guide/api-mediation/api-mediation-smf.md?plain=1#L3

Reply:
I updated sentence. Please, take a look.